packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -36,6 +73,20 @@ const NameType = (props) => {
   const isRegex = (string) =>
     string.charAt(0) === '/' && string.charAt(string.length - 1) === '/';
 
+  const isTypeUnion = (strType) => {
+    return typeof strType === 'string' && strType === 'Union';
+  };
```

Comment:
```
const isTypeUnion = (strType) => strType === 'Union'
```
It's safe to skip the typeOf check, even if it's `undefined` the check will still return `false`